README.md

```diff
@@ -135,6 +135,7 @@ This Tool returns `data` with following format
 | -------------- | --------- | ------------------------------- |
 | file           | `object`  | Uploaded file data. Any data got from backend uploader. Always contain the `url` property |
 | caption        | `string`  | image's caption                 |
+| withCaption    | `boolean` | add caption to image            |
```

Comment:
I think it's redundant property since we already have "caption" field that can be empty. 

Developer can disable caption for all image blocks using tool config. It should not affect output data